README.md

```diff
@@ -1,15 +1,20 @@
 # tic-tac-toe
 
- This is a collaborative project about building a simple tic-tac-toe game. it is basically about
+ This is a collaborative project about building a simple tic-tac-toe game. it is basically
   applying object oriented programming concepts in ruby using classes, instances and methods to create the interface and game logic
  
- The game is comprises a minimum and maximum of two players. Each player chooses a separate character X or O. The board has 9 spaces of which 3 are horizonal, vertical and 2 diagonal. when a player uses his/her character to fill up three spaces, either diagonal, horizontal or vetical before the other, a winner emerges and the game restarts if there's no winner(a tie), the game ends and restarts
+ The game comprises a minimum and maximum of two players. Each player chooses a separate character X or O. The board has 9 spaces of which 3 are horizonal, vertical and 2 diagonal. when a player uses his/her character to fill up three spaces, either diagonal, horizontal or vetical before the other, a winner emerges and the game restarts if there's no winner(a tie), the game ends and restarts
```

Comment:
This description is not very correct. The board has 9 spaces, right. ... of which 3 are horizontal, vertical and 2 diagonal is not right.

A better way to describe it will be 9 spaces with 3 rows and 3 columns. A player wins the game if he fills up an entire row, column or diagonal with his own character.

You may also consider showing this with small images of the board. (Optional)